OpenRA.Mods.Common/TraitsInterfaces.cs

```diff
@@ -158,6 +158,8 @@ public interface INotifyProduction { void UnitProduced(Actor self, Actor other,
 	public interface INotifyOtherProduction { void UnitProducedByOther(Actor self, Actor producer, Actor produced, string productionType, TypeDictionary init); }
 	public interface INotifyDelivery { void IncomingDelivery(Actor self); void Delivered(Actor self); }
 
+	public interface INotifyMineLaying { void MineLaying(Actor self, CPos location); void MineLaid(Actor self, Actor mine); }
```

Comment:
This should get a `[RequireExplicitImplementation]`.

Reply:
Done. Plus rebased on top of bleed.